INSTALLATION.md

```
@@ -145,6 +148,28 @@ npm install
 
 The prerequisites are now installed. The next step will be to get the app up and running.
 
+# Installation using Docker
+
+If you prefer to use Docker, you can install the app using the following command:
+
+## Build the Docker Image:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)